activesupport/lib/active_support/cache/strategy/local_cache.rb

```diff
@@ -102,7 +102,7 @@ def decrement(name, amount = 1, options = nil) # :nodoc:
 
         protected
           def read_entry(key, options) # :nodoc:
-            if local_cache
+            if local_cache = local_cache()
```

Comment:
Nice. Clarify by assigning to a local var with a different name than the reader method, even just `if cache = local_cache`